WorkloadManagementSystem/Executor/JobScheduling.py

```diff
@@ -49,14 +49,14 @@ def optimizeJob( self, jid, jobState ):
         3. Production jobs are sent directly to TQ
         4. Check if staging is necessary
     """
-    #Reschedule delay
+    # Reschedule delay
```

Comment:
All changes but one are beautifying... It would be good if all developers could use the conventions adopted long ago